### PR TITLE
[BUG] Audio Issues when Rotating Device

### PR DESCRIPTION
**Pull request by ahammer**
## Purpose

- Bluetooth was cutting out
- Speaker was jumping back to Bluetooth

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

**Review by ShaunaSong — DISMISSED** [review]

**Review by jamchengms — COMMENTED** [review]
one minor thing i noticed in code

val audioManager = context.getSystemService(Context.AUDIO_SERVICE) as AudioManager
audioManager is initialized in isHeadsetActive() instead of start

does it make sense we only set the audioManager once in the start() instead?

**Comment by ahammer** [comment]
> one minor thing i noticed in code
> 
> val audioManager = context.getSystemService(Context.AUDIO_SERVICE) as AudioManager audioManager is initialized in isHeadsetActive() instead of start
> 
> does it make sense we only set the audioManager once in the start() instead?

Deleted, it comes from constructor, didn't need it there.

**Review by jamchengms — DISMISSED** [review]

**Review by lpeixin — DISMISSED** [review]

File: azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/manager/AudioSessionManager.kt [hunk 1]
```diff
@@ -61,19 +62,23 @@ internal class AudioSessionManager(
 
     suspend fun start() {
 
+        // On first launch we need to init the redux-state, check Bluetooth and Headset status
+        if (!initialized) {
+            initializeAudioDeviceState()
+            updateBluetoothStatus()
```
Comment (phileo99) [inline azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/manager/AudioSessionManager.kt:68]: is `updateBluetoothStatus()` guaranteed to be called after `onServiceConnected()`?

Comment (ahammer) [reply azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/manager/AudioSessionManager.kt:68]: Yeah maybe, I'll look. In testing it didn't switch on rotation, but might be undesirable.

Comment (ahammer) [reply azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/manager/AudioSessionManager.kt:68]: @phileo99  Actually, I might do this differently.

Going to retest. UpdateBluetoothStatus needs to be called at different times, e.g. after service, or after permissions granted. (first launch vs other launches).  

I'm going to change the fix and test again.

**Review by iaulakh — APPROVED** [review]

**Review by jamchengms — APPROVED** [review]

**Review by phileo99 — APPROVED** [review]
tested, ran Espresso UI tests.
LGTM
👍

**Pull request merged**